### PR TITLE
Issue a warning instead of a syntax error for blif delay constraints

### PR DESCRIPTION
**Pull request by Coloquinte**
Fix issue #4152

### CODE REVIEW

File: frontends/blif/blifparse.cc [hunk 1]
```diff
@@ -256,6 +256,16 @@ void parse_blif(RTLIL::Design *design, std::istream &f, IdString dff_name, bool
 				continue;
 			}
 
+			if (!strcmp(cmd, ".area") || !strcmp(cmd, ".delay") || !strcmp(cmd, ".wire_load_slope") || !strcmp(cmd, ".wire") ||
+			    !strcmp(cmd, ".input_arrival") || !strcmp(cmd, ".default_input_arrival") || !strcmp(cmd, ".output_required") ||
+			    !strcmp(cmd, ".default_output_required") || !strcmp(cmd, ".input_drive") || !strcmp(cmd, ".default_input_drive") ||
+			    !strcmp(cmd, ".max_input_load") || !strcmp(cmd, ".default_max_input_load") || !strcmp(cmd, ".output_load") ||
+			    !strcmp(cmd, ".default_output_load"))
+			{
+				log_warning("Blif delay constraints (%s) are not supported.");
```
Comment (gatecat) [inline frontends/blif/blifparse.cc:265]: the format argument for the `%s` seems to be missing here

**Comment by Coloquinte** [comment]
I'm getting a CI error with a mismatch on a test (specifically [gcc-11 c++-11](https://github.com/Coloquinte/yosys/actions/runs/7624650671/job/20767254455)), but it seems to be already present [on master](https://github.com/Coloquinte/yosys/actions/runs/7624645149/job/20767236034)

**Comment by mmicko** [comment]
Please rebase on master. GH actions are fixed meanwhile

**Comment by Coloquinte** [comment]
> Please rebase on master. GH actions are fixed meanwhile

It seems that my branch is already based on latest master (2f9fcc2)

**Comment by Coloquinte** [comment]
The issue persists, and apparently it happens [on other pull requests](https://github.com/YosysHQ/yosys/actions/runs/7620657653/job/20756552244). Any idea where this may come from?

**Comment by Coloquinte** [comment]
The Icarus version that is used is the latest (0db1a0c). Given the message (`Your branch is up to date with 'origin/master'.`), it seems vars.IVERILOG_VERSION is indeed set to `master`. I don't know which version is supposed to work.

**Comment by mmicko** [comment]
192b6aec96fde982e6ddcb28b346d5893aa8e874 is  required version of iverilog

**Comment by Coloquinte** [comment]
Somehow the latest master is pulled. Is the Github variable set to 192b6aec96fde982e6ddcb28b346d5893aa8e874?

**Comment by mmicko** [comment]
Well it looks like variables are not working and are actually taken from PR person context which is not good in this case. Will put version in actions file

**Comment by Coloquinte** [comment]
Very weird behaviour 😮 Thank you for the debugging

**Pull request merged**